src/base/bittorrent/session.cpp

```diff
@@ -1124,6 +1124,7 @@ void Session::configure(libtorrent::settings_pack &settingsPack)
             }
         }
 
+        settingsPack.set_str(libt::settings_pack::outgoing_interfaces, networkInterface().toStdString());
```

Comment:
Yes, but the loop exists for a specific purpose which has nothing to do with the outgoing interface. So I did not want to admix that into the loop.